inlong-dashboard/src/pages/GroupDetail/DataStream/index.tsx

```diff
@@ -178,11 +184,14 @@ const Comp = ({ inlongGroupId, readonly, mqType }: Props, ref) => {
             <Button type="link" onClick={() => onDelete(record)}>
               {t('basic.Delete')}
             </Button>
-            {record?.status && (record?.status === 120 || record?.status === 130) && (
-              <Button type="link" onClick={() => onWorkflow(record)}>
-                {t('meta.Stream.ExecuteWorkflow')}
-              </Button>
-            )}
+            {record?.status &&
+              ((groupStatus === 130 && record?.status === 100) ||
+                groupStatus === 120 ||
+                groupStatus === 130) && (
```

Comment:
Could you please move those two lines into one?

Reply:
An error occurs when merging：
![image](https://user-images.githubusercontent.com/88174078/202067349-1dae09db-3a2c-406c-b177-dcdc1dc07dab.png)